Source/JavaScriptCore/runtime/NativeFunction.h

```diff
@@ -105,25 +53,29 @@ struct TaggedNativeFunctionHash {
 static_assert(sizeof(NativeFunction) == sizeof(void*));
 static_assert(sizeof(TaggedNativeFunction) == sizeof(void*));
 
+static inline TaggedNativeFunction toTagged(NativeFunction function)
+{
+    return function.retagged<HostFunctionPtrTag>();
+}
+
 } // namespace JSC
 
 namespace WTF {
 
 inline void add(Hasher& hasher, JSC::NativeFunction key)
 {
-    add(hasher, bitwise_cast<uintptr_t>(key));
+    add(hasher, key.executableAddress());
```

Comment:
We can't hash the signed value?

Reply:
`executableAddress()` is the signed value.